test/test.js

```diff
@@ -5,6 +5,7 @@ if (!chai) {
   var chai = require('chai');
   var chaiImmutable = require('../chai-immutable');
   var Immutable = require('immutable');
+  var otherImmutable = require('../node_modules/immutable/dist/immutable.min.js');
 
   chai.use(chaiImmutable);
   typeEnv = 'Node.js';
```

Comment:
Another way to do that, somewhat cleaner and most likely compatible with PhantomJS, but probably a bit more difficult to achieve, is to deep clone/duplicate `Immutable` right before `var assert = chai.assert;` and assign this copy to your `otherImmutable` (that could be renamed `clonedImmutable` accordingly).

Then I believe you can remove all those `if (typeEnv === 'Node.js')` and let the runners test these.

Would you mind exploring this idea before we can merge?

Thanks!